core/src/test/java/org/apache/servicecomb/core/exception/ExceptionsTest.java

```diff
@@ -64,8 +72,15 @@ void should_convert_unknown_server_exception_to_invocation_exception() {
     assertThat(invocationException).hasCause(exception);
     assertThat(invocationException.getStatus()).isEqualTo(INTERNAL_SERVER_ERROR);
     assertThat(invocationException.getErrorData()).isInstanceOf(CommonExceptionData.class);
-    assertThat(Json.encode(invocationException.getErrorData()))
-        .isEqualTo("{\"code\":\"SCB.50000000\",\"message\":\"Unexpected exception when processing none. msg\"}");
+
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
objectMapper no need create in each test case.

Reply:
Should I create a static variable available to the entire test class like this?

![image](https://github.com/user-attachments/assets/c4eda876-b16f-4155-a15c-c9f681a69592)

Reply:
yes, this will be reuseable